samples/test/samples.test.js

```diff
@@ -26,9 +27,22 @@ const compute = new Compute();
 describe('samples', () => {
   describe('quickstart', () => {
     const name = `gcloud-ubuntu-${uuid.v4().split('-')[0]}`;
+
     after(async () => deleteVM(name));
-    it('should run the quickstart', () => {
+    it('should run the quickstart', async () => {
       const output = execSync(`node quickstart ${name}`);
+      const externalIP = execSync(`gcloud compute instances describe ${name} \
+    --format='get(networkInterfaces[0].accessConfigs[0].natIP)' --zone=us-central1-c`)
+        .toString('utf8')
+        .trim();
+      try {
+        await waitOn({
+          resources: [externalIP],
+        });
+      } catch (err) {
+        console.log(err);
```

Comment:
I don't think we should have the `try` or `catch` here (unless I'm missing something).  If this throws, I assume the test is broken and we should fail CI.  Right?

Reply:
I was hoping that, if it fails, we can still potentially continue with the test without damaging the integrity of the test itself, because we're only waiting to see if the VM is available. But, it might be safe to assume that if we can't ping the VM, the test itself will also fail; happy to remove it.